src/SingletonPaymasterV6.sol

```diff
@@ -51,14 +50,14 @@ contract SingletonPaymasterV6 is BaseSingletonPaymaster, IPaymasterV6 {
     function validatePaymasterUserOp(
         UserOperation calldata userOp,
         bytes32 userOpHash,
-        uint256 maxCost
+        uint256 requiredPreFund
```

Comment:
Could we also update the variable name in IPaymaterV6 from `maxCost` to `requiredPreFund`